model/src/cfd.rs

```diff
@@ -319,7 +324,7 @@ impl Order {
     /// Check the oracle event ID for sanity
     ///
     /// An id within [25h, 23h] from now is considered sane.
-    fn is_oracle_event_id_sane_to_take(&self, now: OffsetDateTime) -> bool {
+    fn is_oracle_settlement_time_sane(&self, now: OffsetDateTime) -> bool {
```

Comment:
Nit: the oracle doesn't know about `settlement` but when it will attest to something. 
Also, it's not about the oracle but about the oracle's event. Maybe better something like: `is_oracle_event_timestamp_sane`

Reply:
I suggested this so I'll reply.

I think this is the taker's interpretation of the maker's offer. It reasons about the data in there and knows what certain things mean. One of those things is the `oracle_event_id`, which, in the context of this offer, represents the time at which the oracle will allow the non-collaborative "settlement" of the CFD we are intending to open.

I think your suggestion is more precise, but to understand that function name you need to know more about certain implementation details. At a high level, I think the question we want to answer here is: "does the offer being considered have a satisfactory (sane) time-to-live". But maybe "oracle settlement time" is not all that clear.